wgpu-types/src/lib.rs

```diff
@@ -883,6 +883,15 @@ bitflags::bitflags! {
         ///
         /// This is a native only feature.
         const SHADER_INT64 = 1 << 55;
+        /// Allows shaders to use i64 and u64 atomics.
+        ///
+        /// Supported platforms:
+        /// - Vulkan
+        /// - DX12 (DXC only)
```

Comment:
Should maybe mention the VK extension required, and the DXC shader model/feature/thing? Idk how widespread support is / how new this is.